util/util.js

```diff
@@ -138,12 +144,32 @@ function exitRoom(lobby, roomId, playerId) {
     //Notify the player that they left the room
     var connection = lobby.getPlayerConnection(playerId)
     connection.send(new events.NotificationEvent('left-room').convertToJSONString())
-    return undefined
+    return true
+}
+
+function getLobbyStats(lobby) {
+    var stats = {}
+    stats.maxRoomsPerLobby = lobby.MAX_ROOMS_PER_LOBBY
+    stats.numFullRooms = lobby.fullRooms.size
+    stats.numAvailableRooms = lobby.availableRooms.size
+    stats.membersInAvailableRooms = 0
+    stats.membersInFullRooms = 0
+    if (lobby.availableRooms.size > 0) {
+        lobby.availableRooms.forEach((value) => { stats.membersInAvailableRooms += value.roomMembers.length })
+    }
+    if (lobby.fullRooms.size > 0) {
+        lobby.fullRooms.forEach((value) => { stats.membersInFullRooms += value.roomMembers.length })
+    }
+    stats.numOfConnectedClients = Object.keys(lobby.connectedClients).length
+    stats.membersStillInLobby = (stats.numOfConnectedClients - (stats.membersInAvailableRooms + stats.membersInFullRooms))
+    stats.lobbyRoomsOccupancy = ((1 - (lobby.rooms.size / lobby.MAX_ROOMS_PER_LOBBY)) * 100).toFixed(2) + '%'
```

Comment:
Is lobby.rooms.size the number of occupied rooms? If so, I think occupancy should just be:

```
(lobby.rooms.size / lobby.MAX_ROOMS_PER_LOBBY) * 100).toFixed(2) + '%'
```

Reply:
Removed lobbyRoomsOccupancy for now, it's not used in the stats at least now.
Will add in the future with the correct calculation you mentioned.